### PR TITLE
Add confocal images

### PR DESCRIPTION
**Pull request by weiglszonja**
The histology data is in Olympus Image File format which includes a main setting file (.oif) and an associated directory with data and setting files (.tif, .bmp, .txt, .pty, .roi, and .lut).

I'm using the [oiffile](https://pypi.org/project/oiffile/) package to load the image and metadata to write the images to NWB.
Currently I'm using the name and version of the software (e.g. FLUOVIEW FV1000, 1.2.6.0) and the depth information.
Currently I'm creating an `Images` container and adding a `GrayscaleImage` per file (24 files in total) with a name and description using the extracted metadata. 

@CodyCBakerPhD 
I'm not sure if there is an extension for storing all relevant metadata that can be extracted from the setting files, if not, in the future it might be worth creating one.

### CODE REVIEW

File: src/tye_lab_to_nwb/neurotensin_valence/images/neurotensinconfocalimagesdatainterface.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        super().__init__(file_path=file_path)
+
+    def get_metadata(self) -> dict:
+        metadata = super().get_metadata()
+
+        metadata_from_file = dict()
+        settings = self.oif.filesystem.settings
+
+        # Add session_start_time
+        session_start_time = settings["Acquisition Parameters Common"]["ImageCaputreDate"]
```
Comment (CodyCBakerPhD) [inline src/tye_lab_to_nwb/neurotensin_valence/images/neurotensinconfocalimagesdatainterface.py:46]:
```suggestion
        session_start_time = settings["Acquisition Parameters Common"]["ImageCaptureDate"]
```

Comment (CodyCBakerPhD) [reply src/tye_lab_to_nwb/neurotensin_valence/images/neurotensinconfocalimagesdatainterface.py:46]: Unless it is spelled that way in the settings?

**Comment by CodyCBakerPhD** [comment]
> I'm not sure if there is an extension for storing all relevant metadata that can be extracted from the setting files, if not, in the future it might be worth creating one.

I'm not familiar with any, but yeah eventually would be a great thing to have!

**Comment by CodyCBakerPhD** [comment]
The example file seems to have only the images per 6-depth

So for the final aggregated .oif file, that's not being written yet?

File: src/tye_lab_to_nwb/neurotensin_valence/images/neurotensinconfocalimagesdatainterface.py (hunk 1 above)
Comment (weiglszonja) [reply src/tye_lab_to_nwb/neurotensin_valence/images/neurotensinconfocalimagesdatainterface.py:46]: I had to check twice as well, but yes 😄 
Screenshot 2023-03-27 at 11 39 51

**Pull request merged**